deepchem/trans/tests/test_transformers.py

```diff
@@ -69,6 +69,43 @@ def load_unlabelled_data():
   return loader.featurize(input_file)
 
 
+def test_featurization_transformer():
+  fp_size = 2048
+  tasks, all_dataset, transformers = load_delaney('Raw')
+  train = all_dataset[0]
+  transformer = FeaturizationTransformer(
+      transform_X=True,
+      dataset=train,
+      featurizer=dc.feat.CircularFingerprint(size=fp_size))
+  new_train = transformer.transform(train)
+
+  assert new_train.y.shape == train.y.shape
+  assert new_train.X.shape[-1] == fp_size
+
+
+def test_DAG_transformer():
+  """Tests the DAG transformer."""
+  np.random.seed(123)
+  tf.random.set_seed(123)
+  n_tasks = 1
+
+  # Load mini log-solubility dataset.
+  current_dir = os.path.dirname(os.path.abspath(__file__))
+  featurizer = dc.feat.ConvMolFeaturizer()
+  tasks = ["outcome"]
+  input_file = os.path.join(current_dir,
+                            "../../models/tests/example_regression.csv")
+  loader = dc.data.CSVLoader(
+      tasks=tasks, smiles_field="smiles", featurizer=featurizer)
+  dataset = loader.create_dataset(input_file)
+  transformer = dc.trans.DAGTransformer(max_atoms=50)
+  dataset = transformer.transform(dataset)
+  # The transformer generates n DAGs for a molecule with n
+  # atoms. These are denoted the "parents"
+  for idm, mol in enumerate(dataset.X):
+    assert dataset.X[idm].get_num_atoms() == len(dataset.X[idm].parents)
```

Comment:
It could be good to run a list comprehension that generates a boolean array, and then check at the end with one `assert` statement that everything is `True`.

Reply:
This code was cruft leftover in this PR and was merged in an earlier PR, so the changes went away in the rebase. It might be good to do a later pass to simplify some of these tests though!